java/src/jmri/jmrix/loconet/SlotManager.java

```diff
@@ -840,6 +840,9 @@ public final void setAcceptAnyLACK() {
     /**
      * Handles OPC_LONG_ACK replies to programming slot operations.
      *
+     * LACK 0x6D00 which requests a retransmission is handled
```

Comment:
Only a comment, with no code changes?  Seens odd...

Reply:
It took me a half-hour to figure out that LACK 0x6D00 was handled outside the routine that's labelled as handling all LACKs.  The comment is so that nobody else has to do that.